### ISSUE TITLE
Mac builds

### ISSUE THREAD

**Issue by Moosems** [Futura-Py/TimerX#111]
See my repo [here](https://github.com/Moosems/PyInstallerBuild) for how to set up mac builds

**Comment by im-coder-lg** [comment]
I need instructions to implement this.

**Comment by im-coder-lg** [comment]
Can you tell how?

**Comment by Moosems** [comment]
Yep, so first of all we need to set up for pyinstaller. You can copy/paste DIP.spec and then edit all mentions of DIP but pretty much fill in the blank and I can look at it later. After that it's just setting up a release with the tag `latest-mac-build`. It will update the dmg in there and should work and be backwards compatible
